setup.py

```diff
@@ -36,8 +36,7 @@
         'Environment :: Console',
         'Intended Audience :: Information Technology',
         'License :: OSI Approved :: Apache Software License',
-        'Operating System :: POSIX',
-        'Operating System :: MacOS',
+        'Operating System :: OS Independent',
```

Comment:
Again quite independent from the PR title

Reply:
True. However, the integration test in this PR allowed me to test it under Windows.